pt/lessons/libraries/poolboy.md

```diff
@@ -110,24 +109,50 @@ end
 
 ## Usando Poolboy
 
-Agora que temos o nosso `Worker`, podemos testar o Poolboy. Vamos criar um módulo simples que cria processos simultâneos usando a função `:poolboy.transaction`:
+Agora que temos o nosso `PoolboyApp.Worker`, podemos testar o Poolboy. Vamos criar um módulo simples que cria processos simultâneos usando o Poolboy. `:poolboy.transaction/3` é a função que usamos para interagir com o poll de gerenciadores. Crie `lib/poolboy_app/test.ex`:
 
 ```elixir
-defmodule Test do
+defmodule PoolboyApp.Test do
   @timeout 60000
 
   def start do
-    tasks =
-      Enum.map(1..20, fn i ->
-        Task.async(fn ->
-          :poolboy.transaction(:worker, &GenServer.call(&1, {:square_root, i}), @timeout)
-        end)
-      end)
-
-    Enum.each(tasks, fn task -> IO.puts(Task.await(task, @timeout)) end)
+    1..20
+    |> Enum.map(fn i -> async_call_square_root(i) end)
+    |> Enum.each(fn task -> await_and_inspect(task) end)
   end
+
+  defp async_call_square_root(i) do
+    Task.async(fn ->
+      :poolboy.transaction(
+        :worker,
+        fn pid -> GenServer.call(pid, {:square_root, i}) end,
+        @timeout
+      )
+    end)
+  end
+
+  defp await_and_inspect(task), do: task |> Task.await(@timeout) |> IO.inspect()
 end
 ```
-Se você não tiver gerenciadores de pool disponíveis, Poolboy chegará ao tempo limite após o período de tempo limite padrão (cinco segundos) e não aceitará nenhuma nova solicitação. Em nosso exemplo, aumentamos o tempo limite padrão para um minuto para demonstrar como podemos alterar o valor de tempo limite padrão.
 
-Mesmo que estamos tentando criar vários processos *(total de vinte no exemplo acima)* a função `:poolboy.transaction` limitará o total de processos criados a cinco *(mais dois gerenciadores de estouro, se necessário)* como definimos em nossa configuração. Todos os pedidos serão tratados pelo grupo de gerenciadores em vez de criar um novo processo para cada pedido.
\ No newline at end of file
+Execute a função de teste para ver o resultado.
+
+```shell
+$ iex -S mix
+```
+
+```elixir
+iex> PoolboyApp.Test.start()
+process #PID<0.182.0> calculating square root of 7
+process #PID<0.181.0> calculating square root of 6
+process #PID<0.157.0> calculating square root of 2
+process #PID<0.155.0> calculating square root of 4
+process #PID<0.154.0> calculating square root of 5
+process #PID<0.158.0> calculating square root of 1
+process #PID<0.156.0> calculating square root of 3
+...
+```
+
+Se você não tiver gerenciadores de pool disponíveis, Poolboy chegará ao tempo limite após o período de tempo limite padrão (cinco segundos) e não aceitará nenhuma nova solicitação. Em nosso exemplo, aumentamos o tempo limite padrão para um minuto para demonstrar como podemos alterar o valor de tempo limite padrão. No caso desse app, você pode abservar o error se você mudar o de `@timeout` para menos de 1000.
```

Comment:
Typo at the end of the sentence: "... pode observar o **erro** se você **mudar o `@timeout`para** ..."